airflow/dags/parse_rt/create_service_alerts_external_table.yml

```diff
@@ -0,0 +1,134 @@
+operator: operators.ExternalTable
```

Comment:
So I'm starting to think about making these things either a task before dbt, or dbt pre-hooks, for a couple reasons.

1. It lets us run drop/replace _when we are also building the stage model on top_. My main worry right now is that we may drop an external table, hit a failure re-creating (unsure if we can do this in a transaction), and then the dbt DAG won't succeed.
2. I agree about not running hourly, in retrospect it's just not necessary.

I do _not_ think we should do once-only, because that leaves us vulnerable to stale code. If our long-term plan is to _always_ clean and stage data, then we should define external tables in the dbt project, I think. It also means that changes to external tables are executed alongside their stage tables in the tightest matter possible.

Reply:
I am open to pre-hooks. I saw this: https://github.com/dbt-labs/dbt-external-tables referenced in the [dbt docs on hooks & operations](https://docs.getdbt.com/docs/building-a-dbt-project/hooks-operations), assume we might use something like it. 

I do think that basically we should just document whatever we do to make the conceptual model clear. I think the topline point of my comment was that having these execute whenever the data is scraped and specifically having the external creation tasks **as Airflow tasks dependent on data scraping** is clearly conceptually wrong.

Reply:
Yes totally agree.

I don't think we can use that plugin unfortunately, as we need the Python client for setting hive partition options.

Can I move the tasks to the `transform_warehouse` DAG for now, and maybe we investigate using https://github.com/fal-ai/fal in the future?

Reply:
Ahhh right I keep forgetting about the pesky Python hive business. Until we implement as pre-hooks I would probably just say one more time that a dedicated external tables DAG just for visibility might make sense. But if you feel strongly about sticking it in `transform_warehouse` I am ok with that.

Also, the existing once-only `gtfs_schedule_history` DAG is external table creation (that's why I originally proposed once-only), and so it should probably follow whatever we do here. If we think that `fal` implementation is imminent, maybe we leave it as-is, but if we think that whatever you do here (dedicated DAG or move into `transform_warehouse`) is going to be the state for a while I think we should have a ticket to make `gtfs_schedule_history` follow that same pattern. And maybe we should check with @j-meelah and @mjumbewu about what is done in the Payments pipeline.

Reply:
Moved to the dbt DAG.